src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.tsx

```diff
@@ -146,11 +146,9 @@ export const ContactTags: React.FC<ContactTagsProps> = ({
               <Autocomplete
                 multiple
                 freeSolo
-                autoSelect
                 autoHighlight
                 fullWidth
                 loading={loading}
-                popupIcon={<ContactTagIcon />}
                 filterSelectedOptions
                 value={tagList}
```

Comment:
Can you explain this change to me? Seems like we're removing functionailty from the whole site for one page

Reply:
We have been removing `autoSelect` throughout the codebase as we come across it because it is not user-friendly (it selects things that the user did not click on).
`popupIcon` is not currently doing anything on this component. The icon does not show anywhere.